examples/language-features/instances.qnt

```diff
@@ -1,37 +1,27 @@
-// This example demonstrates support for TLA+ INSTANCE in Quint
-module InstanceEx {
+module Instances {
   module A {
```

Comment:
I'll make a pass removing nested modules from all examples once we are removing nested modules :smile: unit tests will require some work, since I'm wrapping modules inside a module in many tests.